fixtures/eurolite/led-bar-3-hcl-bar.json

```
@@ -0,0 +1,303 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Bar-3 HCL Bar",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Rainer Jung"],
+    "createDate": "2020-01-13",
```

Could you please update this to today?

```suggestion
    "createDate": "2020-01-14",
```